### PR TITLE
add Popmotion aframe-role + D3 links

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,14 +44,20 @@ Third-party A-Frame components (of the entity-component system).
 
 - [aframe-layout](https://github.com/ngokevin/aframe-layout)
 
-### React
+### Adapters and Plugins for Frameworks and Libraries
```
Comment (ngokevin) [inline README.md]: hmm a bit lengthy. maybe just `With Frameworks and Libraries`?

then move webpack into it as well

**Comment by ngokevin** [comment]
r+wc

File: README.md (hunk 1 above)
Comment (cvan) [inline README.md]: yeah

**Comment by ngokevin** [comment]
would also add the d3 example

File: README.md (hunk 1 above)
Comment (cvan) [inline README.md]: updated

**Comment by cvan** [comment]
rr? @ngokevin

**Comment by ngokevin** [comment]
r+

**Pull request merged**